packages/thumbprint-react/components/ServiceCard/index.jsx

```diff
@@ -1,32 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Image from '../Image/index.jsx';
 import styles from './index.module.scss';
 
-class ServiceCardImage extends React.Component {
-    componentDidMount() {
-        // These imports are only needed client-side and allow for lazy-loading images. They should
-        // be changed to `import()` once Gatsby 2 launches. We're currently limited by the version
-        // of Webpack that Gatsby uses.
-        // https://github.com/gatsbyjs/gatsby/issues/461
-        // https://github.com/thumbtack/thumbprint-archive/issues/960
-        /* eslint-disable global-require */
-        require('lazysizes/plugins/unveilhooks/ls.unveilhooks');
-        require('lazysizes');
-        // `ls.attrchange.js` re-renders the image when the props change:
-        // https://github.com/aFarkas/lazysizes/issues/339
-        require('lazysizes/plugins/attrchange/ls.attrchange.js');
-        /* eslint-enable */
-    }
-
-    render() {
-        const { url, alt } = this.props;
-
-        return (
-            <div className={`${styles.image} lazyload`} data-bg={url} role="img" aria-label={alt} />
-        );
-    }
-}
+const ServiceCardImage = ({ url, sources, alt }) => (
+    <Image
+        className={styles.image}
+        sources={sources}
+        containerAspectRatio={8 / 5}
```

Comment:
Should we enforce this? Don't have strong feelings but if a user uses a different aspect ratio they may not notice the placeholder is incorrect. E.g, they request "16:9" but that "8/5" is what's in the placeholder it'll still "work" but just the placeholder will be wrong.

Reply:
Hm, I say we should keep it since it makes the lazy-loading placeholder work. We'd otherwise have to add a prop to `ServiceCardImage` that is the aspect ratio of the image which seems like overkill since we decided when designing the component that we only want to support `8:5` images.

Reply:
Will the user have to add `aspectRatio="8:5"` to `ImageServicePicture`? If that's the case my question was why not let `containerAspectRatio` get auto-populated. 

The concern was that the user might add `aspectRatio="16:9"` (for an experiment) which, unless they were paying close attention, would appear to work but in fact the placeholder size is incorrect. So unless I'm missing something it's not really enforcing an aspect ratio.

Reply:
The functionality of the aspect ratio in this PR is the same as it was in the existing version of `ServiceCardImage`. Both versions use CSS to enforce an aspect ratio. If a user were to pass in a `1:1` image, `ServiceCardImage` would still render it as `8:5` with `object-fit: cover`.